tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,8 @@
+configuration-properties.names=deployer.org, \
```

Comment:
I didn't see you used names at first.
Now that we're there, we might as well remove: enable-random-app-name-prefix, app-name-prefix, disk, bp ... and everything that does not apply to tasks